crates/cac_client/src/interface.rs

```diff
@@ -176,12 +176,19 @@ pub extern "C" fn cac_get_last_modified(client: *mut Arc<Client>) -> *const c_ch
 #[no_mangle]
 pub extern "C" fn cac_get_config(
     client: *mut Arc<Client>,
-    query: *const c_char,
+    filter_query: *const c_char,
+    filter_prefix: *const c_char,
 ) -> *const c_char {
-    let filter = if query.is_null() {
+    null_check!(
```

Comment:
INFO: moved [this check](https://github.com/juspay/superposition/pull/127/files#diff-1cc6f99f841b0d796e99630ff9386809a7735358480a55c7888c492a13c59581L202) here